packages/widgetbook/lib/src/workbench/renderer.dart

```diff
@@ -17,22 +17,23 @@ class Renderer extends StatelessWidget {
     BuildContext context, {
     required List<WidgetbookAddOn> addons,
     WidgetbookAddOn? multiPropertyAddon,
-    required int index,
   }) {
     return MultiProvider(
+      key: ValueKey(useCaseBuilder),
       providers: [
         if (multiPropertyAddon != null)
-          multiPropertyAddon.providerBuilder(context, index),
+          multiPropertyAddon.providerBuilder!(context),
         ...addons
             .where((element) => element != multiPropertyAddon)
-            .map((e) => e.providerBuilder(context, 0))
+            .map((e) => e.providerBuilder!(context))
```

Comment:
Forcing this value to be non-null looks like a potential bug and problem to me. Why is`providerBuilder` nullable?